charts/px-backup/values.yaml

```
@@ -0,0 +1,60 @@
+namespace: px-backup
+
+customeRegistryEnabled: false
+dockerRegistryURL: docker.io
+dockerImageRepoName: portworx
+imagePullSecrets: docregistry-secret
+imagePullPolicy: Always
```

These all should go under one section
```suggestion
image:
  registry: docker.io
  repo: portworx
  pullSecrets: docregistry-secret
  pullPolicy: Always
```
Don't need `customRegistryEnabled`. Also `pullSecrets` should be an array, since multiple secrets can be added

---

Moved all registry, repo, pullSecret and pullPolicy inside images section.
We still need `customRegistryEnabled: false` flag because all images are not in portworx repo like keycloak, MySQL so based on this flag we are either pulling this images from respective repo's , if custom registry is enabled then images will have registry and repo mentioned in values.yaml